.github/workflows/e2e-test.yml

```
@@ -0,0 +1,26 @@
+name: E2E Test in Docker
+
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, reopened]
```

If you use both `opened` and `synchronize`, CI will trigger twice when you open a new PR. If you open it with a label, that's one extra trigger.

```suggestion
    types: [synchronize, reopened]
```